pulp_ansible/app/settings.py

```diff
@@ -20,5 +20,5 @@
     "dynaconf_merge": True,
 }
 
-ANSIBLE_API_HOSTNAME = "http://" + socket.getfqdn()
+ANSIBLE_API_HOSTNAME = "https://" + socket.getfqdn()
```

Comment:
when reusing this PR, don't forget this, CI was failing before this change (it doesn't come from template)

Reply:
FYI @dkliban @pulp/ansible